DataConnectors/CyberArk/CyberArk Data Connector.json

```diff
@@ -0,0 +1,104 @@
+{
+    "id": "CyberArk",
+    "title": "CyberArk Enterprise Password Vault (EPV) Syslog",
+    "publisher": "Cyber-Ark",
+    "descriptionMarkdown": "CyberArk Enterprise Password Vault generates an xml Syslog messages for every action taken against the Vault.  The EPV will send the xml messages through the Sentinel.xsl translator to be converted into CEF standard format and sent to a syslog staging server of your choice (syslog-ng, rsyslog). The Log Analytics agent installed on your syslog staging server will import the messages into Azure Log Analytics",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "CyberArkSyslog",
+            "baseQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Cyber-Ark\"\n| where DeviceProduct == \"Vault\"\n"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "CyberArk Alerts",
+            "query": "\nCommonSecurityLog\n| where LogSeverity == \"7\" or LogSeverity == \"10\"\n| sort by TimeGenerated desc"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "CommonSecurityLog",
```

Comment:
The data type name should be "CommonSecurityLog (CyberArk)"  with values in the parenthesis same as the value in the legend name.